src/content/agda/editor-note.tex

```diff
@@ -4,7 +4,7 @@
 \newcommand{\optlang}{Agda\xspace}
 \newcommand{\contributors}{\urlref{https://formalverification.io}{formalverification.io}\xspace}
 \newcommand{\optlangfig}{fig/icons/agda.png}
-\newcommand{\optlangcode}{\srcsnippet{content/3.6/code/agda/snippet03.agda}{purple}{agda}}
+\newcommand{\optlangcode}{\srcsnippet{content/3.6/code/agda/snippet03.agda}{black}{agda}}
```

Comment:
In the `editor-note.tex` file, it says that the color of the line next to the code snippets will match the primary color of the language's official logo.  The Agda logo is black and white, so I guess black is the best option... which actually appears as gray in the pdf output.